src/main/java/cooking/ReceipePrompt.java

```diff
@@ -0,0 +1,41 @@
+package cooking;
+
+import java.util.Scanner;
+
+public class ReceipePrompt {
+
+    private float amount;
```

Comment:
These two are temporary fields (will be set and the vlaues will change out of control, might even be default.